docs/development/releasing.rst

```diff
@@ -396,6 +416,12 @@ The procedure for this is straightforward:
 
 #. Repeat the above steps for the astropy-helpers, using the same version series.
 
+#. Create an issue titled "Release final vX.Y".
```

Comment:
does this issue needed?

Reply:
@astrofrog wanted it. 🤷‍♀

Reply:
oh, sure. but that was 3 years ago and he didn't complained since of not doing it. I'm OK either way, but feel it's superfluous, we never forgot to release even without having an issue.

Reply:
Tom R, what say you?

Reply:
Remove it! 🔥